content/en/docs/tasks/administer-cluster/declare-network-policy.md

```diff
@@ -10,6 +10,9 @@ This document helps you get started using the Kubernetes [NetworkPolicy API](/do
 {{% /capture %}}
 
 {{% capture prerequisites %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
 You'll need to have a Kubernetes cluster in place, with network policy support. There are a number of network providers that support NetworkPolicy, including:
```

Comment:
Remove this item and the list of network providers. The k8s cluster requirement is taken care of in the prerequisites include, and any functioning cluster includes a network provider. You could add an item that says something about making sure you've configured a network provider, and link to the appropriate topic, however.